src/input.c

```diff
@@ -1088,13 +1088,12 @@ void process_mouse_over(int x, int y)
 				if (subWindow == NULL)
 					break;
 
-				ebx = ebx & 0xFFFFFF00;
 				edi = cursorId;
 				esi = (int)subWindow;
 				// Not sure what this is for, no windows actually implement a handler
 				// RCT2_CALLFUNC_X(subWindow->event_handlers[WE_UNKNOWN_0E], &eax, &ebx, &ecx, &edx, &esi, &edi, &ebp);
 				cursorId = edi;
-				if ((ebx & 0xFF) != 0)
+				if (0)
```

Comment:
@duncanspumpkin 
Is it likely we will find out what WE_UNKNOWN_0E means if we haven't already?
All the window code is done...